messages/index.js

```diff
@@ -221,11 +229,12 @@ module.exports = {
     nl: "{{#def.n}}moet de eigenschap{{#def.propPlural}} {{=e.params.deps}} bevatten als {{=e.params.property}} is gedefinieerd",
     pl: "{{#def.n}}powinien zawierać pol{{#def.propPlural}} {{=e.params.deps}} kiedy pole {{=e.params.property}} jest obecne",
     "pt-BR":
-      "{{#def.n}} deve ter propriedade{{#def.propPlural}} {{=e.params.deps}} quando a propriedade {{=e.params.property}} estiver presente",
+        "{{#def.n}} deve ter propriedade{{#def.propPlural}} {{=e.params.deps}} quando a propriedade {{=e.params.property}} estiver presente",
```

Comment:
This and othe spaces need removed.

Reply:
The same. Error at the build stage.

Reply:
You need just run this command:

```bash
npm run prettier:write
```

After that commit changes and push them here.